src/controllers/connection/__tests__/helpers.ts

```diff
@@ -3,7 +3,7 @@ import { pino } from 'pino'
 
 import sinon from 'sinon'
 import { Env } from '../../../env.js'
-import type { ILogger } from '../../../logger.js'
+import type { BasicLogger } from '../../../logger.js'
```

Comment:
why the switch? ILogger better represents the interface no?

Reply:
It did not work being as part of `express.Request`